x-pack/plugins/index_management/public/application/sections/home/data_stream_list/data_stream_list.tsx

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import { RouteComponentProps } from 'react-router-dom';
+import { FormattedMessage } from '@kbn/i18n/react';
+import { EuiTitle, EuiText, EuiSpacer, EuiEmptyPrompt } from '@elastic/eui';
+import { ScopedHistory } from 'kibana/public';
+
+import { SectionError, SectionLoading, Error } from '../../../components';
+import { useLoadDataStreams } from '../../../services/api';
+import { DataStreamTable } from './data_stream_table';
+
+interface MatchParams {
+  dataStreamName?: string;
+}
+
+export const DataStreamList: React.FunctionComponent<RouteComponentProps<MatchParams>> = ({
+  match: {
+    params: { dataStreamName },
+  },
+  history,
+}) => {
+  const { error, isLoading, data: dataStreams, sendRequest: reload } = useLoadDataStreams();
+
+  let content;
+
+  if (isLoading) {
+    content = (
+      <SectionLoading>
+        <FormattedMessage
+          id="xpack.idxMgmt.dataStreamList.loadingIndexTemplatesDescription"
+          defaultMessage="Loading data streams…"
+        />
+      </SectionLoading>
+    );
+  } else if (error) {
+    content = (
+      <SectionError
+        title={
+          <FormattedMessage
+            id="xpack.idxMgmt.dataStreamList.loadingIndexTemplatesErrorMessage"
+            defaultMessage="Error loading data streams"
+          />
+        }
+        error={error as Error}
+      />
+    );
+  } else if (Array.isArray(dataStreams) && dataStreams.length === 0) {
+    content = (
+      <EuiEmptyPrompt
+        iconType="managementApp"
+        title={
+          <h1 data-test-subj="title">
```

Comment:
[nit] I think we've typically added a description to the empty prompt as well.

Reply:
Great idea! Done.

![image](https://user-images.githubusercontent.com/1238659/84068687-3649c880-a97e-11ea-9509-108206abb69c.png)